anndata/tests/test_base.py

```diff
@@ -467,8 +467,13 @@ def test_pickle():
 
 
 def test_to_df_dense():
-    df = adata_dense.to_df()
-    df = adata_dense.to_df(layer="test")
+    X_df = adata_dense.to_df()
+    layer_df = adata_dense.to_df(layer="test")
+
+    np.testing.assert_array_equal(adata_dense.layers["test"], layer_df.values)
+    np.testing.assert_array_equal(adata_dense.X, X_df.values)
+    pd.testing.assert_index_equal(X_df.columns, layer_df.columns)
+    pd.testing.assert_index_equal(X_df.index, layer_df.index)
```

Comment:
did this make it in here by accident?

Reply:
Nah, just leaving the code better than I found it. This was throwing a flake8 error (assigning to variable that's never used) and instead of renaming the variables and leaving a bad test I made the test more useful.